test/README.md

```diff
@@ -0,0 +1,43 @@
+# libE57Format Testing
+
+Testing uses the [googletest](https://github.com/google/googletest) framework. The documentation for it may be [found here](https://google.github.io/googletest/).
+
+## Turning Testing On
+
+To turn testing on, set the CMake option `E57_BUILD_TEST` to ON.
+
+## Testing Data
+
+Currently, the testing data is found in another repo: [libE57Format-test-data](https://github.com/asmaloney/libE57Format-test-data).
+
+As we build out testing, the number & size of the test data might grow very quickly. For this reason, the test data is kept separate from the main repo. This gives us some flexibility depending on how large the data set becomes. If ends up being too big for a git repo, for example, we can make it available as a zip download somewhere. It will also let us manage the data in CI better so it doesn't have to be downloaded every time CI is run.
+
```

Comment:
Another advantage of a tarball approach is less git submodule and (possibly?) git LFS wrangling.  Test data doesn't tend to change from one commit to the next.

Reply:
Yes I agree. The drawback is managing the tarball hosting, etc.. Using a repo (for now) makes storage and tracking changes/providing an interface into it easy. I also like having it all available though one site (GitHub).

If we start using it heavily the data will grow beyond a git repo, however, which is why I set it up this way. Easy to switch it out.

(My upload speed is not so great, so managing multiple gigabytes is going to be painful. 😄)